config/imx8ulp.toml

```diff
@@ -0,0 +1,21 @@
+version = [1, 0]  # use simple file write
```

Comment:
Need to add the following message in the commit message:

```
    Add imx8ulp toml file to support sof-imx8ulp.ri binary build.
```

Reply:
Sure